modules/ROOT/pages/clauses/load-csv.adoc

```diff
@@ -397,4 +570,367 @@ If `LOAD CSV` is invoked with a `file:///` URL that points to your disk `file()`
+
+.Comparison: import nodes and relationships together VS splitting them
+====
+.https://data.neo4j.com/importing-cypher/books.csv
+[source, csv]
+----
+id,title,author,publication_year,genre,rating,still_in_print,last_purchased
+19515,The Heights,Anne Conrad,2012,Comedy,5,true,2023/4/12 8:17:00
+39913,Starship Ghost,Michael Tyler,1985,Science Fiction|Horror,4.2,false,2022/01/16 17:15:56
+60980,The Death Proxy,Tim Brown,2002,Horror,2.1,true,2023/11/26 8:34:26
+18793,Chocolate Timeline,Mary R. Robb,1924,Romance,3.5,false,2022/9/17 14:23:45
+67162,Stories of Three,Eleanor Link,2022,Romance|Comedy,2,true,2023/03/12 16:01:23
+25987,Route Down Below,Tim Brown,2006,Horror,4.1,true,2023/09/24 15:34:18
+----
+
+.Bad practice -- Create nodes and relationships at once
+[source, cypher]
+----
+LOAD CSV WITH HEADERS
+FROM 'https://data.neo4j.com/importing-cypher/books.csv'
+AS row
+MERGE (b:Book {id: row.id})
+SET b.title = row.title
+MERGE (a:Author {name: row.author})
+MERGE (a)-[:WROTE]->(b)
+----
+
+.Good practice -- Separate nodes and relationships creation
+[source, cypher]
+----
+// Create `Book` nodes
+LOAD CSV WITH HEADERS
+FROM 'https://data.neo4j.com/importing-cypher/books.csv'
+AS row
+MERGE (b:Book {id: row.id})
+SET b.title = row.title;
+
+// Create `Author` nodes
+LOAD CSV WITH HEADERS
+FROM 'https://data.neo4j.com/importing-cypher/books.csv'
+AS row
+MERGE (a:Author {name: row.author});
+
+// Create `WROTE` relationships
+LOAD CSV WITH HEADERS
+FROM 'https://data.neo4j.com/importing-cypher/books.csv'
+AS row
+MATCH (a:Author{name: row.author})
+MATCH (b:Book{id: row.id})
+MERGE (a)-[:WROTE]->(b);
+----
+
+====
+
+
+=== Avoid `Eager` operators
+
+Depending on the exact shape of your Cypher query, the database may have to craft a query plan that contains the link:https://neo4j.com/docs/cypher-manual/current/planning-and-tuning/operators/operators-detail/#query-plan-eager[`Eager`] operator.
+You may discover if this is the case by link:https://neo4j.com/docs/cypher-manual/current/query-tuning/#how-do-i-profile-a-query[profiling your queries], and attempt tweaking them to avoid it in case.
+
+
+=== Database heap and memory
+
+To help handle larger volumes of transactions, there are a few memory configuration settings you can tweak:
+
+* link:{neo4j-docs-base-uri}/operations-manual/{page-version}/configuration/configuration-settings/#config_server.memory.heap.initial_size[`server.memory.heap.initial_size`] and link:{neo4j-docs-base-uri}/operations-manual/{page-version}/configuration/configuration-settings/#config_server.memory.heap.max_size[`server.memory.heap.max_size`]: set to at least 4G.
+* link:{neo4j-docs-base-uri}/operations-manual/{page-version}/configuration/configuration-settings/#config_server.memory.pagecache.size[`server.memory.pagecache.size`]: ideally, a value large enough to keep the whole database in memory.
+
+As a rule of thumb, you can create or update one million records in a single transaction per 2 GB of heap.
```

Comment:
@gem-neo4j can you verify this statement? (it stems from the getting started guide)

Reply:
I have no clue 🙈

Reply:
commented out for now